deploy/demo.sh

```diff
@@ -188,8 +199,14 @@ demo::start-dind-cluster
 if [[ ${INJECT_LOCAL_IMAGE:-} ]]; then
   demo::inject-local-image
 fi
+if [[ ${DEPLOY_LOG_CONTAINER:-} == 2 ]]; then
```

Comment:
Replace this and the next check in case you decide to go with text values for deployment type.